pennylane/tape/tape.py

```diff
@@ -1089,47 +1089,36 @@ def specs(self):
 
         return self._specs
 
-    def draw(self, charset="unicode", wire_order=None, show_all_wires=False, max_length=None):
-        """Draw the quantum tape as a circuit diagram.
-
-        Consider the following circuit as an example:
-
-        .. code-block:: python3
-
-            with QuantumTape() as tape:
-                qml.Hadamard(0)
-                qml.CRX(2.3, wires=[0, 1])
-                qml.Rot(1.2, 3.2, 0.7, wires=[1])
-                qml.CRX(-2.3, wires=[0, 1])
-                qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
-
-        We can draw the tape after construction:
-
-        >>> print(tape.draw())
-        0: ──H──╭C────────────────────────────╭C─────────╭┤ ⟨Z ⊗ Z⟩
-        1: ─────╰RX(2.3)──Rot(1.2, 3.2, 0.7)──╰RX(-2.3)──╰┤ ⟨Z ⊗ Z⟩
-        >>> print(tape.draw(charset="ascii"))
-        0: --H--+C----------------------------+C---------+| <Z @ Z>
-        1: -----+RX(2.3)--Rot(1.2, 3.2, 0.7)--+RX(-2.3)--+| <Z @ Z>
-
-        Args:
-            charset (str, optional): The charset that should be used. Currently, "unicode" and
-                "ascii" are supported.
+    # pylint: disable=too-many-arguments
+    def draw(
+        self,
+        wire_order=None,
+        show_all_wires=False,
+        decimals=None,
+        max_length=100,
+        show_matrices=False,
+    ):
+        """Draw the quantum tape as a circuit diagram. See :func:`~.drawer.tape_text` for more information.
+
+        Keyword Args:
```

Comment:
Note: `Keyword Args` would refer to the expected keyword arguments passed with a signature that has double-asterisk to unpack a dictionary of keyword arguments.

https://pennylane.readthedocs.io/en/stable/development/guide/documentation.html#functions-and-methods
```suggestion
        Args:
```